ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -358,6 +359,10 @@ Rectangle {
         } else if (event.key === Qt.Key_Escape && control.isReply) {
             control.isReply = false
             event.accepted = true
+        } else if (event.key === Qt.Key_Up && getPlainText().trim() == "") {
+            event.accepted = true
+            control.keyUpPress()
```

Comment:
I was considering the entire keyPress event and have the consuming side decide what to do with it, but then went for just exposing the arrow-up key press.

Happy to change this again, not sure how locked-in we are atm with the StatusChatInput API.

Reply:
If you input space (or spaces) or add a new line - edit will be open.
Checked Discord behavior, if there is no characters input - we edit the message, if any character input (including just one space) - text navigation. I think we must check, that there is not text inside textArea

Reply:
Ah, I thought `getPlainText()` is gonna do that, but it seems it's not.
Thank you!

Reply:
trim removes spaces. Just remove it